lib/screen/home/components/home.dart

```diff
@@ -7,8 +8,8 @@ class Home extends StatelessWidget {
     return Column(
       children: <Widget>[
         const SizedBox(height: 30),
-        const Text(
-          'Oi, fulano',
+        Text(
+          'Oi, ${JWTUtils.getUsername()?.toUpperCase()}',
```

Comment:
deixa pro JWTUtils retornar ja uppercase. Tell, dont ask.

Reply:
Discordo um pouco. O utils retorna o dado. A forma de apresentação deve ser decidida em cada local que for ser mostrado. Até pq posso querer mostrar essa mesma info de outro jeito em outro local. Claro que nesse caso não é tão relevante assim